paddle/fluid/inference/api/analysis_config.cc

```diff
@@ -143,6 +145,18 @@ void AnalysisConfig::EnableMKLDNN() {
   Update();
 }
 
+void AnalysisConfig::EnableQuantizer() {
```

Comment:
```
#ifdef PADDLE_WITH_MKLDNN
xxx
#else
 use_quantizer_ = false
#endif
```

Reply:
fixed in https://github.com/PaddlePaddle/Paddle/pull/16396